.travis.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     script:
     - scripts/setup-minikube-for-linux.sh
     - make test CHARTS=enterprise-suite-latest
+    # Only build latest in the daily cron - don't block PRs or master releases.
+    if: type in (cron)
```

Comment:
Can this go higher in the definition?  I thought it applied to the Reactive-sandbox test at first blush.  (Not worth holding anything up for this though.  Ignore as you see fit.)